sunpy/net/helio/tests/test_helio.py

```diff
@@ -12,6 +12,8 @@
     wsdl_retriever,
 )
 
+pytestmark = pytest.mark.filterwarnings('ignore:Unverified HTTPS request is being made')
```

Comment:
why are we making unverified https requests?!

Reply:
🤷 is this something we should fix instead of ignoring?

Reply:
I think we should understand it at least. Could be cret issues with the helio servers I guess?

Reply:
We had to disable the SSL check for helio.

Reply:
I'll open an issue with the traceback

Reply:
> We had to disable the SSL check for helio.

We did? I thought it didn't actually fix it?

Reply:
That was for some instr data.